src/parser.rs

```diff
@@ -2069,6 +2081,10 @@ impl Parser {
             return self.parse_derived_table_factor(Lateral);
         }
 
+        if self.parse_keyword(Keyword::TABLE) {
```

Comment:
done

Reply:
Please ditch the `return` above the `else`? And make the same change in the next return, so that the whole function is one large if-elseif-else expression.

Reply:
Done